ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Block/Cardano.hs

```diff
@@ -30,9 +30,8 @@ import           Cardano.Crypto (RequiresNetworkMagic (..))
 import qualified Cardano.Crypto as Crypto
 import qualified Cardano.Crypto.Hash.Class as CryptoClass
 import           Cardano.Crypto.Raw (Raw)
-import qualified Cardano.Ledger.Api.Era as L
-import qualified Cardano.Ledger.Api.Transition as L
-import qualified Cardano.Ledger.BaseTypes as SL (natVersion)
+import qualified Cardano.Ledger.Api.Era as SL
+import qualified Cardano.Ledger.Api.Transition as SL
```

Comment:
Only if you feel like it: `Cardano.Ledger.Api` is usually imported `as L` in our code (run e.g. `rg 'Cardano.Ledger.Api.* as'`). But we can also defer work here to https://github.com/IntersectMBO/ouroboros-consensus/issues/596.

Reply:
Absolutely! I should have grepped for other occurrences of `Cardano.Ledger.Api.* as`. I'll change it back. I think it's better not to introduce additional inconsistencies.